docs/usage/getting-started/running.md

```diff
@@ -108,6 +108,18 @@ If you combine both of the above then any single config option in the environmen
 
 Note: it's also possible to change the default prefix from `RENOVATE_` using `ENV_PREFIX`. e.g. `ENV_PREFIX=RNV_ RNV_TOKEN=abc123 renovate`.
 
+#### Using `config.js`
+
+If you use a `config.js`, it will be expected to export a configuration via `module.exports`. The value can be either a plain JavaScript object like in this example where `config.js` exports a plain object:
```

Comment:
```suggestion
If you use a `config.js`, it will be expected to export a configuration via `module.exports`.
The value can be either a plain JavaScript object like in this example where `config.js` exports a plain object:
```

Let's use the one sentence per line format here.